angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngFor="let item of itemsList">
+  <div class="item">
+    <div class="item-container">
+      <div class="info">
+        <ng-container *ngIf="item.state === 'open'; else closed">
+          <!-- no-entry -->
+          <span class="icon {{ item.state }}" appOcticon="no-entry"></span>
+        </ng-container>
+        <ng-template #closed>
+          <!-- issue-closed -->
+          <span class="icon {{ item.state }}" appOcticon="issue-closed"></span>
+        </ng-template>
+        <div>
+          <div class="content">
+            <span class="title">{{ item.title }}</span>
+            <span class="inline" *ngFor="let label of item.labels">
+              <span
+                [style.background-color]="colorMap(label.color)"
+                class="label"
+              >
+                {{ label.name }}
+              </span>
+            </span>
+          </div>
+          <div class="meta">
+            #{{ item.number }}
+            <span *ngIf="item.state === 'open'">
+              opened
+              {{ item.created_at | relativeTime }}
+            </span>
+            by
+            <span class="created-by">Daian</span>
```

Comment:
suggestion: This shouldn't stay hard coded. ;) Should be the "user" field, I think, that will give us the name here. (Interfaces for PRs and issues are in the repository/service folder right now)